portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,79 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
+/* IDs */
 
 #greeting-container {
   margin-top: 20px;
 }
+
+#avatar {
+    width: 40%;
+}
+
+#title > * {
+  display: inline;
+}
+
+#title > a {
+  text-decoration: none;
+}
+
+#introduction{
+    background-color: #FAEBD7;
+}
+
+#education{
+    background-color: #F5DEB3;
+}
+
+#experience{
+    background-color: #CEB088;
+}
+
+#projects{
+    background-color: #967A76;
+}
+
+#fun-stuff{
+    background-color: #534C5A;
+    color: #534C5A;
+}
+
+#fun-stuff:hover{
+  color: white;
+}
+
+/* classes */
+.section {
+  display: inline-block;
+  padding: 1% 20% 1%;
+  background-color: wheat;
+}
+
+.title-icon {
+  width: 3em;
+  padding-left: 1em;
```

Comment:
The meaning of padding is for spacing inside an element, while margin is for spacing between elements. Even though a lot of them time they end up having similar behavior, the semantics are different. In this case, it looks like this should be margin-left.